src/App.js

```diff
@@ -1,16 +1,27 @@
 import './App.css';
-import {Router , Route} from "react-router-dom"
+import {Routes, Route} from "react-router-dom"
+import {useState, useEffect} from 'react';
+import SignUp from './Pages/SignUp';
+import LogIn from './Pages/LogIn';
+import Dashboard from './Pages/Dashboard'
 
-function App() {
-  return (
-    <div>
-      <header>
-      </header>
-      <main>
-      <p> code works </p>
-      </main>
-    </div>
-  );
-}
+export default function App() {
+
+    const [assets, setAssets] = useState([])
+    const [authUser, setAuthUser] = useState(null)
 
-export default App;
\ No newline at end of file
+    return (
+        <div>
+            <main>
+                <Routes>
+                    <Route path="/signup"
+                        element={<SignUp setAuthUser={setAuthUser} />} />
+                    <Route path="/login"
+                        element={<LogIn authUser={authUser} setAuthUser={setAuthUser} />} />
+                    <Route path="/dashboard"
+                        element={<Dashboard authUser={authUser} assets={assets} />} />
```

Comment:
Is this meant to be a protected route? If so you'll want `element={authUser && <Dashboard />}`.

Reply:
it will, have not sorted out the protected on the server side just yet